docs/bsp.md

```diff
@@ -857,14 +975,54 @@ trait TestReport {
 }
 ```
 
-The `TestReport` notification will be sent as the test execution of build targets completes.
+This request may trigger a compilation or other tasks on the selected build targets.
 
-This request may trigger a compilation on the selected build targets. The server is free to send any
-number of `build/compileReport`, `build/publishDiagnostics` and `build/logMessage` notifications
-during compilation before completing the response.
+The server may send any number of `build/task*`, `build/publishDiagnostics` and `build/logMessage` notifications
+to communicate about tasks triggered by the request to the client.
+
+Subtask notifications:
+
+The server may inform about individual test units in subtask notifications. Individual test notifications should
+include the `TestStarted` and `TestFinished` objects in their params:
```

Comment:
Can you elaborate on "include the `TestStarted` and `TestFinished` objects in their params:"?

Reply:
It is supposed to mean the `data` parameter. I'll add that.